app/controllers/application_controller.rb

```diff
@@ -1,2 +1,25 @@
+require 'rake'
+Rake::Task.clear
```

Comment:
It clears the existing prerequisites, actions, comments, and arguments of a rake task. It is necessary to avoid tasks being loaded several times in dev mode.
On removing this line, it is calling sending previos mails again several times.